src/allmydata/crypto/rsa.py

```diff
@@ -58,31 +62,38 @@ def create_signing_keypair(key_size):
     return priv_key, priv_key.public_key()
 
 
-def create_signing_keypair_from_string(private_key_der):
+def create_signing_keypair_from_string(private_key_der: bytes) -> tuple[PrivateKey, PublicKey]:
```

Comment:
There should be no change of behavior in this function.  All of the changes are to fix complaints mypy has about this function now that it is annotated.